src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt

```diff
@@ -387,9 +377,9 @@ class PrisonerIepLevelReviewServiceTest {
       // When
       prisonerIepLevelReviewService.processOffenderEvent(prisonOffenderEvent)
 
-      // Then - the prisoner was on STD at BXI so they on this level
+      // Then - the prisoner was on BAS at BXI so they should remain on this level
       val expectedPrisonerIepLevel = PrisonerIepLevel(
-        iepCode = "STD",
+        iepCode = "BAS",
```

Comment:
switched to testing using Basic level because it's not the default in this prison meaning this test actually shows the level being preserved and definitely not resetting to a default